api/types/network/network.go

```diff
@@ -112,13 +112,13 @@ type ConfigReference struct {
 }
 
 var acceptedFilters = map[string]bool{
+	"dangling": true,
 	"driver":   true,
 	"type":     true,
```

Comment:
oh, wait; wanted to sort the list alphabetically; let me order this one as well 👍